### PR TITLE
SAK-31990 Fix for NPE while accessing saveMesage from outside sakai context

### PR DESCRIPTION
**Pull request by chaitut715**
https://jira.sakaiproject.org/browse/SAK-31990

### CODE REVIEW

**Comment by bjones86** [comment]
I think this is okay provided there are no opponents to exposing the 2nd method publicly.

@chaitut715, for future reference, your commits should be prefixed with the JIRA ticket #; ie SAK-31990 in this case. You can either edit your commit message and force push, or whomever merges this can do it for you via squash + merge.

@ncaidin, do we have a CLA on file for Kumar?

**Comment by chaitut715** [comment]
@bjones86 , Is commit message fine now.

**Comment by bjones86** [comment]
@chaitut715, no you haven't pushed anything new; the commit message is unchanged.

**Comment by chaitut715** [comment]
@bjones86 , updated commit message.

File: msgcntr/messageforums-api/src/java/org/sakaiproject/api/app/messageforums/MessageForumsMessageManager.java [hunk 1]
```diff
@@ -40,7 +40,17 @@
...
     public void saveMessage(Message message, boolean logEvent);
-    
+    /**
+     * 
+     * @param message message
+     * @param logEvent logEvent
+	 * @param toolId id of the forums tool
+	 * @param userId user id
+	 * @param contextId context id'
+	 *
```
Comment (bjones86) [inline msgcntr/messageforums-api/src/java/org/sakaiproject/api/app/messageforums/MessageForumsMessageManager.java]: Just noticed the spacing looks off in here. Sorry should have mentioned this before. Please use whatever is used or most prevalent in the file, be it tabs or spaces or whatever strange mixture of the two...

**Comment by chaitut715** [comment]
@bjones86 Please check now

**Comment by ncaidin** [comment]
Yes, a very easy way to see. Go to http://licensing.apereo.org/  .  List of all contributors and institutions with license agreements on file is there.

I don't see one for "Kumar Ch". 

For information on submitting a license agreement please see  section 2 of https://www.apereo.org/licensing  , "Inbound licensing" . There will be a link to a pdf file (ICLA). Review it, fill it out, sign it, and then email the completed form to licensing@apereo.org  .

If you are working for an institution, the institution (company, vendor, school, etc.) that institution also needs a contributor license agreement called a CCLA (corporate contributor license agreement). 

Two different forms.

**Comment by bjones86** [comment]
Great, thanks for that link @ncaidin.

@chaitut715 you'll have to submit a CLA and possibly a CCLA (if you work for an institution). We can only take your PR after the forms have been filed.

**Comment by chaitut715** [comment]
@bjones86  @ncaidin , I work for an organization which uses Sakai.
I am not doing any office work w.r.t Sakai now.
It's my personal interest to contribute to sakai.
So please help me out whether I need to submit ICLA or CCLA?

**Comment by ncaidin** [comment]
Hi @chaitut715 , if you are not doing office work for Sakai and it is on your own time, an ICLA should be sufficient.

**Comment by ncaidin** [comment]
ICLA is now on file.

**Comment by bjones86** [comment]
Thanks @ncaidin

**Pull request merged**